docs/install/install-cave.md

```diff
@@ -93,7 +95,7 @@ This method is recommended for personal use and requires Administrative priviled
 1. Download and install 64-bit [Miniconda Python3.7 version 4.8.2 for Windows](https://repo.anaconda.com/miniconda/Miniconda3-py37_4.8.2-Windows-x86_64.exe)
     - **Allow** Miniconda3 to set **PATH** and other environment variables
     - **Register** miniconda as the default python
-    ![Miniconda options](../images/minicondaOptions.png)
+    ![Miniconda options](../images/minicondaOptions.png)s
```

Comment:
Should this "s" be there?

Reply:
I wish I could say I did that on purpose, to test our code review process (which worked really well!), but that was a mistake that I didn't notice.  Good catch!  I updated this branch with that fix, so you shouldn't see it now when you look at the "files changed".